### ISSUE TITLE
Keystroke to trigger language selection in new document

### ISSUE THREAD

**Issue by PeterWone** [microsoft/vscode#133655]
This business is awesome

![image](https://user-images.githubusercontent.com/5498936/134464031-bbb16c69-25ad-4fe9-be29-6c342bbb3690.png)

It would be _more_ awesome if I didn't have to grab the mouse to use it. If there is already a keystroke equivalent to clicking _Select a language_ the link text would be a pretty good place to tell people what it is.

**Comment by IllusionMH** [comment]
It's regular Change Language Mode (`workbench.action.editor.changeLanguageMode`) that on Windows has <kbd>Ctrl</kbd>+<kbd>K</kbd> <kbd>M</kbd> keybinding.

**Comment by PeterWone** [comment]
Thank you! But I still think it would add value to say so in the link.

**Comment by TylerLeonhardt** [comment]
So you're saying something like?:

"Select a language (<kbd>Ctrl</kbd>+<kbd>K</kbd> <kbd>M</kbd>) to get started. Start typing to dismiss, or don't show this again."

**Comment by PeterWone** [comment]
Yes, exactly. Hardly even needs testing, massively improves discoverability. Biggest cost is localisation.

**Comment by TylerLeonhardt** [comment]
@misolori @isidorn what are your thoughts?

**Comment by isidorn** [comment]
@TylerLeonhardt we had this previously as a way to teach users a keybinding.
However it looked more dense, and users were hitting the wrong keybinding `Ctrl + k, Ctrl + M`.

Bottom line: I am fine either way. I would probably stick with how it is for clarity

**Comment by PeterWone** [comment]
A discoverability issue is never going to get 20 votes, those who need it don't know they need it and those who don't, don't care anymore.

**Comment by misolori** [comment]
I'd also at least add it as a tooltip since we do add keybindings in those as well

**Comment by TylerLeonhardt** [comment]
> I'd also at least add it as a tooltip since we do add keybindings in those as well

I think this is a reasonable compromise of having the information be available but also not cluttering.

**Comment by PeterWone** [comment]
Personally I always thought `Ctrl+K`, `Ctrl+M` is a clumsy onscreen notation that belongs in print.

I would have chosen `^K` `^M` had anyone asked me. It has a pedigree considerably longer than most of the people likely to argue about it, and it makes clear the difference between `^K` `^S` and `^K` `S` .

**Comment by coderpen-me** [comment]
Hi, can i make this change @TylerLeonhardt

**Comment by TylerLeonhardt** [comment]
Feel free to give it a shot and submit a PR

**Comment by Suven-p** [comment]
@TylerLeonhardt Would a PR be accepted if this was a backlog candidate instead of backlog?

**Comment by TylerLeonhardt** [comment]
The milestone doesn't impact whether or not we would accept a PR. The only things that really impact a PR going in is:

* the quality of the code (the assignee/reviewer has final say in that)
* the availability of the assignee/reviewer to review your PR (be patient with us 🙂)

I'll review your PR as soon as I can.
